external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -218,3 +226,35 @@ private[kafka010] class KafkaMicroBatchStream(
     }
   }
 }
+
+object KafkaMicroBatchStream extends Logging {
+
+  /**
+   * Compute the difference of offset per partition between latestAvailablePartitionOffsets
+   * and latestConsumedPartitionOffsets.
```

Comment:
Mismatch between `latestConsumedOffset` and `latestConsumedPartitionOffsets` in doc and code.

Reply:
I changed it to "partition offsets in the latestConsumedOffset" to make it more clear.